src/javascripts/components/tabs.js

```diff
@@ -0,0 +1,80 @@
+;(function (global) {
```

Comment:
The structure and name spacing looks great, thanks @alex-ju 👍  I would prefer for the classes to be defined as variables to avoid repetition but we can discuss our preferred convention at dev time.

Reply:
I know, not my best code, but I expect to revisit this one as we'll have a `tabs` component in `govuk-frontend` too